rel/overlay/bin/remsh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
Are there any bashisms in this script? https://linux.die.net/man/1/checkbashisms can help.

I’d prefer (but not hard requirement) to restrict to `sh` syntax, so this works by default on BSD et.al. systems.

Reply:
```bash
$ checkbashisms ~/couchdb/rel/overlay/bin/remsh
$ echo $?
0

```

Reply:
Ah, there is a single one if I force it:

```
$ checkbashisms -n -x -p -f couchdb/rel/overlay/bin/remsh
possible bashism in couchdb/rel/overlay/bin/remsh line 69 (alternative test command ([[ foo ]] should be [ foo ])):
if [[ ! -z "$VERBOSE" ]]; then
```

I'll PR this up and auto-merge since it's a trivial fix.